modules/README.md

```diff
@@ -40,12 +40,15 @@ Modules that are still in development and less stable are labeled with the
 * **[htcondor-execute-point]** ![community-badge] ![experimental-badge] :
   Manages a group of execute points for use in an [HTCondor
   pool][htcondor-configure].
+* **[pbspro-execution]** ![community-badge] ![experimental-badge] :
```

Comment:
nit: Probably a little late in the game to bring this up but almost every other module (maybe all) are a noun. `pbspro-execution` sounds like a verb to me. Would `pbspro-executor` be a better fit, or `pbspro-execution-host(s)`.

Reply:
The PBS nomenclature is:

- Server hosts
- Client / Submit hosts (sometimes I see Submission)
- Execution hosts

I prefer the brevity to adding "host" to everything but we can consider changes while experimental.